src/com/lftechnology/java/training/alina/basic/stemleaf/DataSet.java

```diff
@@ -0,0 +1,32 @@
+package com.lftechnology.java.training.alina.basic.stemleaf;
+
+/**
+ * Class DataSet consists of data points to display stem and leaf
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class DataSet implements Comparable<DataSet> {
+	public int dataSet;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make dataSet a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename field "dataSet" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1700)